indico/modules/events/abstracts/controllers/management.py

```diff
@@ -57,6 +57,18 @@ def _process(self):
         return RH._process(self)
 
 
+class RHManageAbstractBase(AbstractMixin, RHManageAbstractsBase):
+    """Base class for all abstract management RHs"""
+
+    def _checkParams(self, params):
+        RHManageAbstractsBase._checkParams(self, params)
+        AbstractMixin._checkParams(self)
+
+    def _checkProtection(self):
+        RHManageAbstractsBase._checkProtection(self)
+        AbstractMixin._checkProtection(self)
```

Comment:
So basically, only the event managers can access the abstract management pages, which is satisfied by the `_checkProtection` of `RHModificationBaseProtected`, meaning that I'll have to remove the `AbstractMixin._checkProtection(self)` line.